src/portable/status.rs

```diff
@@ -388,13 +389,24 @@ async fn get_remote_async(instances: Vec<String>) -> anyhow::Result<Vec<RemoteSt
     Ok(result)
 }
 
+async fn get_cloud(cloud_client: &CloudClient) -> anyhow::Result<Vec<RemoteStatus>> {
+    match crate::cloud::ops::list(cloud_client).await {
+        Ok(rv) => Ok(rv),
+        Err(e) if e.is::<TimeoutError>() => {
+            print::warn("Timed out getting cloud instances.");
```

Comment:
I would also print the error after the table. Things before the table can be unnoticed, and then users will wonder why they don't see some instances.

So the stragegy as I see it: return normal `Result` here. And carry it until the table is printed then if there was the error, print it and exit with erroneous status code.